### PR TITLE
MULTI-122: Initial commit for kubefed docs (multicluster)

### PR DESCRIPTION
**Pull request by abrennan89**
- added kubefed directory
- added understanding kubefed content
- added attributes doc for kubefed
- added topic map entry
- added symlinks for images and modules

### CODE REVIEW

File: modules/kubefed-document-attributes.adoc [hunk 1]
```diff
@@ -0,0 +1,26 @@
+// Standard document attributes to be used in the documentation
+//
+// The following are shared by all documents:
+:toc:
+:toclevels: 4
+:experimental:
+//
+// Product content attributes, that is, substitution variables in the files.
+//
+:product-title: OpenShift Container Platform
```
Comment (bergerhoffer) [inline modules/kubefed-document-attributes.adoc]: If you're planning on including the "include::modules/common-attributes.adoc[]" in all of your assemblies (which it looks like the assembly in this PR does), I wouldn't redefine product-title in this attribute file too.

**Review by bergerhoffer — COMMENTED** [review]
@abrennan89 I put the versions for 4.1 and 4.2 - just wanted to confirm that that's correct!

File: modules/kubefed-document-attributes.adoc [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+:ProductName: OpenShift KubeFed
```
[the remark this answers is not in the document]
Comment (ousleyp) [reply modules/kubefed-document-attributes.adoc]: paging @vikram-redhat 

@bergerhoffer brings up some great points here (thanks, Andrea!). I think we should make sure to use a CNV-specific name when we switch to two different variables (IIRC, we were most recently planning to use `:ProductNameStart:` and `:ProductName:`).

**Comment by abrennan89** [comment]
Thanks @bergerhoffer , I've made some updates so this should be ready now :slightly_smiling_face:

**Pull request merged**